tests/compiler/__snapshots__/attributes.test.ts.snap

```diff
@@ -430,6 +430,19 @@ exports[`attributes static attributes on void elements 1`] = `
 }"
 `;
 
+exports[`attributes static attributes with backslash or backtick 1`] = `
+"function anonymous(app, bdom, helpers
+) {
+  let { text, createBlock, list, multi, html, toggler, comment } = bdom;
+  
+  let block1 = createBlock(\`<div foo=\\"\\\\\\\\a\\" bar=\\"\\\\\\\\n\\" baz=\\"\\\\\`\\"/>\`);
```

Comment:
lol at this

Reply:
JS string that contains a JS function that contains a JS string => backslashes are escaped 3 times => 2^3 backslashes per backslash. Very cool.